docs/eigenda/operator-guides/run-a-node/registration.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+
+# Register Your Operator
+
+Your operator will not begin receiving traffic from the EigenDA disperser until it has registered for one or more quorums with EigenDA. Note, as discussed in [delegation requirements](../requirements/delegation-requirements/), that registration with an EigenDA quorum requires that an operator already be [registered as an operator with EigenLayer](https://docs.eigenlayer.xyz/operator-guides/operator-installation) and to have a minimum amount of stake delegated within each quorum to be registered. 
+
+## Opt-in to an EigenDA Quorum
+
+If you meet the delegation requirements for opting into one or more quorums, you can execute the following command from the `eigenda-operator-setup/mainnet` folder to opt-in to the desired quorums:
```

Comment:
reference to `eigenda-operator-setup/mainnet` folder must be made within a dedicated 'mainnet' tab, with a separate folder referenced for Holesky.

As of 4/22, we only support dual quorum on Holesky and all references to dual quorums must be removed from mainnet docs.

Reply:
we can just add here that `dual-quorum` is only supported for mainnet